version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "2.2-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "2.6-SNAPSHOT"
```

Comment:
The thing is: If you want things to build here (travis), then you actually have to publish the snapshot version. But if you published it and later on realise you forgot something in the code, you can't publish the same snapshot version again (it will not be picked up locally on your development machine). So you have to increase the snapshot version. And the release version should always relate directly to the snapshot version. As such some releases versions are skipped.